Src/java/qdm/schema/qdm.5.3.xsd

```diff
@@ -272,6 +272,7 @@
 					<xs:element name="method" type="dt:Code" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="result" type="dt:Any" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="component" type="Component" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="relatedTo" type="dt:Code" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Yep, I verified the one in the model info but missed it in the xsd. Since this has pull request has already been merged, I'll consider this a typo and make the change directly.